web/src/main/java/no/nav/foreldrepenger/fordel/web/app/rest/journalføring/ManuellJournalføringMapper.java

```diff
@@ -56,44 +56,11 @@ static StatusDto mapFagsakStatusTilStatusDto(FagsakStatusDto fagsakStatusDto) {
         };
     }
 
-    static String tekstFraBeskrivelse(String beskrivelse) {
```

Comment:
Trenger du ikke denne for oppgaver som hentes fra Gosys? 
Det kan fortsatt potensielt dukke opp oppgaver i Gosys som er overført på sidelinjen uten at fordel vet om dem. Vi får vite om endret tema på journalpost - men ikke at oppgavene flyttes.

Reply:
Den brukes ikke i frontend lenger - mulig man ville se hele teksten.